addons/mail/models/mail_mail.py

```diff
@@ -248,6 +251,14 @@ def send(self, auto_commit=False, raise_exception=False):
                 email sending process has failed
             :return: True
         """
+
+        # Raise or return out of method to ensure we are not inadvertently sending emails
+        if not db_whitelisted(self.env.cr.dbname):
+            if self.env['ir.config_parameter'].sudo().get_param('mail.email_db_whitelist_fail_silent'):
```

Comment:
The logic here is to ensure compatibility for some views that rely on this method returning True, eg: Send Message from document chatter